src/StackAdmin/Storage.Management/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -49,76 +49,55 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <!-- We Moved this to a project we build: Microsoft.Azure.Management.Storage.6.4.0-preview -->
+    <Reference Include="Microsoft.Azure.KeyVault.Core">
+      <HintPath>..\..\..\..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
+    <!--    <Reference Include="Microsoft.Azure.Management.Storage, Version=6.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
You could shorten this if you wanted the reference version.

Reply:
done